client/src/pages/Admin/EditDetails/index.js

```diff
@@ -81,12 +93,16 @@ const EditDetails = () => {
   };
 
   useEffect(() => {
-    if (adminOrg.id) {
-      setFormData({
-        organisationName: adminOrg.organisationName,
-        uniqueSlug: adminOrg.uniqueSlug,
-      });
+    async function setOrganisation() {
```

Comment:
```suggestion
    function setOrganisation() {
```
no need for async keyword here since you didn't use await keyword inside the function